apps/crn-server/test/handlers/working-group/update-deliverables-handler.test.ts

```diff
@@ -1,31 +1,36 @@
+import { WorkingGroupDeliverable } from '@asap-hub/model';
 import { workingGroupUpdateHandler } from '../../../src/handlers/working-group/update-deliverables-handler';
 import {
-  getWorkingGroupEvent,
+  getWorkingGroupSquidexEvent,
+  getWorkingGroupContentfulEvent,
   getWorkingGroupResponse,
 } from '../../fixtures/working-groups.fixtures';
 import { getDataProviderMock } from '../../mocks/data-provider.mock';
-import { WorkingGroupDeliverable } from '@asap-hub/model';
 
-describe('Working Group update handler', () => {
+describe.each`
+  cms             | getEventFunction
+  ${`Squidex`}    | ${getWorkingGroupSquidexEvent}
+  ${`Contentful`} | ${getWorkingGroupContentfulEvent}
+`('Working Group update handler', ({ cms, getEventFunction }) => {
   const workingGroupDataProviderMock = getDataProviderMock();
   const handler = workingGroupUpdateHandler(workingGroupDataProviderMock);
 
   afterEach(() => jest.clearAllMocks());
 
-  test('updates Pending/In Progress deliverable statuses to Not Started/Incomplete if working group is completed', async () => {
+  test(`updates Pending/In Progress deliverable statuses to Not Started/Incomplete if working group is completed for a ${cms} event`, async () => {
     const deliverables: WorkingGroupDeliverable[] = [
       { description: 'A pending deliverable', status: 'Pending' },
       { description: 'An in progress deliverable', status: 'In Progress' },
     ];
-    const event = getWorkingGroupEvent({ complete: true, deliverables });
```

Comment:
we don't really use the value of `complete` and `deliverables`  from the event. We get those values from data provider entry. So I'm removing them from the tests because they don't make a difference in what we're expecting.